ambry-store/src/main/java/com.github.ambry.store/StoreFindToken.java

```diff
@@ -156,10 +223,18 @@ public Offset getOffset() {
     return offset;
   }
 
-  public UUID getSessionId() {
+  UUID getSessionId() {
     return sessionId;
   }
 
+  UUID getIncarnationId() {
+    return incarnationId;
+  }
+
+  boolean getInclusive() {
+    return inclusive == 1 ? true : false;
```

Comment:
1. In general, avoid `expr ? true : false` pattern and simply substitute that with `expr`.
2. Ideally, this check should be `... == (byte) 1` or `(... != 0)`.